core/src/main/java/com/linecorp/armeria/internal/metric/RequestMetricSupport.java

```diff
@@ -145,9 +157,22 @@ private RequestMetricSupport() {}
         Timer totalDuration();
     }
 
+    private static final class DefaultActiveRequestMetrics implements ActiveRequestMetrics {
+        private final AtomicInteger active;
```

Comment:
- How about using `LongAdder` which is known to have less contention?
- How about extending `LongAdder` instead of using it as a field, for reduced footprint?